dashboards/mariadb/mariadb.json

```diff
@@ -0,0 +1,2662 @@
+{
+  "name": "MariaDB Overview",
+  "description": null,
+  "pages": [
+    {
+      "name": "MariaDB Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 4
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![MariaDB logo](https://mariadb.com/wp-content/uploads/2019/11/mariadb-logo-vert_blue-transparent.png)\n\n## Overview\n\n[MariaDB](https://www.mysql.com/) MariaDB Server is one of the most popular open source relational databases. It’s made by the original developers of MySQL and guaranteed to stay open source. It is part of most cloud offerings and the default in most Linux distributions.\n\nIt is built upon the values of performance, stability, and openness, and MariaDB Foundation ensures contributions will be accepted on technical merit. Recent new functionality includes advanced clustering with Galera Cluster 4, compatibility features with Oracle Database and Temporal Data Tables, allowing one to query the data as it stood at any point in the past.\n\n---\n\n## About this dashboard\n\n* For information on the MariaDB on-host integration, please visit the documentation [here](https://docs.newrelic.com/install/mysql/).\n* To view the source code, please visit the [nri-mysql repository](https://github.com/newrelic/nri-mysql).\n* Use the `Node` variable at the top of this page to filter your results."
```

Comment:
Can we shorten this content? In the dashboard screenshot, it looks like this section is automatically getting trimmed.

Reply:
content is shortened